Number_types/include/CGAL/boost_mp.h

```diff
@@ -223,7 +396,19 @@ struct RET_boost_mp <NT, boost::mpl::int_<boost::multiprecision::number_kind_int
 
 template <class NT>
 struct RET_boost_mp <NT, boost::mpl::int_<boost::multiprecision::number_kind_rational> >
-    : RET_boost_mp_base <NT> {};
+    : RET_boost_mp_base <NT> {
+    typedef NT Type;
+    struct To_interval
+        : public CGAL::cpp98::unary_function< Type, std::pair< double, double > > {
+
+        std::pair<double, double> operator()( const Type& x ) const {
+
+          const auto& xnum = boost::multiprecision::numerator(x);
+          const auto& xden = boost::multiprecision::denominator(x);
+          return Boost_MP_internal::to_interval(x, xnum, xden);
```

Comment:
Since `Boost_MP_internal::to_interval` takes its last 2 arguments by value, it would be better to write this body as a single line (`to_interval(x,numerator(x),denominator(x))`), for types where `numerator` returns a value and not a reference.